backend-implicaction/src/main/java/com/dynonuggets/refonteimplicaction/controller/RelationController.java

```diff
@@ -71,4 +71,13 @@ public ResponseEntity cancelRelationByUser(@PathVariable("userId") Long userId1)
         relationService.cancelRelation(userId1, userId2);
         return ResponseEntity.noContent().build();
     }
+
+    @GetMapping(value = "/{senderId}/confirm")
+    public ResponseEntity<RelationsDto> confirmRelation(
+            @PathVariable("senderId") Long senderId
+    ) {
+        final Long receiverId = authService.getCurrentUser().getId();
+        relationService.confirmRelation(senderId, receiverId);
+        return ResponseEntity.noContent().build();
```

Comment:
tu dois récupérer la relation qui a été update et la renvoyer au front : 
```suggestion
        RelationDto relation = relationService.confirmRelation(senderId, receiverId);
        return ResponseEntity.ok(relation);
```